node/CHANGELOG.md

```diff
@@ -1,63 +1,92 @@
-- fix ToolRunner - _getSpawnSyncOptions - [#873](https://github.com/microsoft/azure-pipelines-task-lib/pull/873)
+- Replaced deprecated "sync-request" lib and Added new Async methods - [#932](https://github.com/microsoft/azure-pipelines-task-lib/pull/932)
 
-### 4.0.2
+### 4.5.0
 
-- Updated mockery because of vulnerabilities - [#878](https://github.com/microsoft/azure-pipelines-task-lib/pull/878)
+- Added `execAsync` methods that return native promises. Marked `exec` methods that return promises from the Q library as deprecated [#905](https://github.com/microsoft/azure-pipelines-task-lib/pull/905)
 
-## 4.1.0
+### 4.4.0
 
-Backported from ver.`3.4.0`:
+- Add `getBoolFeatureFlag` [#936](https://github.com/microsoft/azure-pipelines-task-lib/pull/936)
+
+### 4.3.1
+
+- Resolve CVE-2022-24999 in qs 6.9.4 [#924](https://github.com/microsoft/azure-pipelines-task-lib/pull/924)
+
+### 4.3.0
+
+- Described types for `argIf` - [#920](https://github.com/microsoft/azure-pipelines-task-lib/pull/920)
 
-- Include uncought exceptions stack trace to the output logs - [#895](https://github.com/microsoft/azure-pipelines-task-lib/pull/895)
 
-## 4.2.0
+### 4.2.0
 
 - Added unhandledRejection event - [#912](https://github.com/microsoft/azure-pipelines-task-lib/pull/912)
 
-## 4.3.0
+### 4.1.0
 
-- Described types for `argIf` - [#920](https://github.com/microsoft/azure-pipelines-task-lib/pull/920)
+Backported from ver.`3.4.0`:
 
-## 4.3.1
+- Include uncought exceptions stack trace to the output logs - [#895](https://github.com/microsoft/azure-pipelines-task-lib/pull/895)
 
-- Resolve CVE-2022-24999 in qs 6.9.4 [#924](https://github.com/microsoft/azure-pipelines-task-lib/pull/924)
+### 4.0.2
 
-## 4.4.0
+- Updated mockery because of vulnerabilities - [#878](https://github.com/microsoft/azure-pipelines-task-lib/pull/878)
 
-- Add `getBoolFeatureFlag` [#936](https://github.com/microsoft/azure-pipelines-task-lib/pull/936)
+### 4.0.1-preview
 
+- Added node16 to task.schema.json - [#852](https://github.com/microsoft/azure-pipelines-task-lib/pull/852)
+- fix ToolRunner - _getSpawnSyncOptions - [#873](https://github.com/microsoft/azure-pipelines-task-lib/pull/873)
 
-## 4.5.0
+### 4.0.0-preview
 
-- Added `execAsync` methods that return native promises. Marked `exec` methods that return promises from the Q library as deprecated [#905](https://github.com/microsoft/azure-pipelines-task-lib/pull/905)
+- Introduced support for node 16 task handler - [#844](https://github.com/microsoft/azure-pipelines-task-lib/pull/844)
 
-## 4.6.0
+## 3.x
 
-- Replaced deprecated "sync-request" lib and Added new Async methods - [#932](https://github.com/microsoft/azure-pipelines-task-lib/pull/932)
+### 3.4.0
 
-## 4.6.1
+- Updated mockery and mocha dependencies - [#875](https://github.com/microsoft/azure-pipelines-task-lib/pull/875)
+
+- Include uncought exceptions stack trace to the output logs - [#895](https://github.com/microsoft/azure-pipelines-task-lib/pull/895)
```

Comment:
minor typo "uncaught"

Reply:
Sorry, merged before saw your comment, the typo was there [previously](https://github.com/microsoft/azure-pipelines-task-lib/blob/45b9782cdf0b6557b5662c89c52712f6a1ba83c9/node/CHANGELOG.md#410), will fix in next PRs